src/Avatar/Avatar.js

```diff
@@ -22,6 +22,7 @@ function getStyles(props, context) {
       borderRadius: '50%',
       height: size,
       width: size,
+      boxShadow: src ? `inset 0px 0px 0px 1px ${avatar.borderColor}` : 'none',
```

Comment:
Oh, interesting, I compared the current and new implementation, and couldn't see a meaningful difference, but perhaps I'm not testing correctly.